certified-connectors/secplugs/apiDefinition.swagger.json

```diff
@@ -1,12 +1,12 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "Secplugs Scan",
-    "description": "Secplugs Scan for scanning Emails content, files contents and Urls, Secplugs scan is a platform which creates hybrid security platform using various security provider",
+    "title": "Scancloud APIs",
```

Comment:
@Ajaatsecplugs please remove the workdAPI from the title and from the description, it's not allowed to use it. You can check the documentation [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info) and [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title) for more information.

Reply:
Title update is not allowed for deployed connectors, it's a breaking change. Can you please revert the title to Secplugs Scan? If you want a new name you'll have to submit a new PR following [these guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission).